### PR TITLE
Update current.yaml -- fix issue #27201

### PR DESCRIPTION
**Pull request by johnzheng1975**
for not-grpc request, should not get grpc-status -- fix issue https://github.com/envoyproxy/envoy/issues/27201

This is a copy PR from: https://github.com/envoyproxy/envoy/pull/27219
Previous PR is reviewed approved.. But there is some sign-off issue.

Please review again for this PR, thanks.

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Fixes commit #PR or SHA]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by johnzheng1975** [comment]
@yanavlasov, please help to review, I added a runtime feature protection per your request. Thanks.
image
image

**Comment by johnzheng1975** [comment]
@wbpcode  @ohadvano , FYI

File: changelogs/current.yaml [hunk 1]
```diff
@@ -78,6 +78,11 @@ bug_fixes:
     Fixed the bug that updating :ref:`scope_key_builder
     <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.ScopedRoutes.scope_key_builder>`
     of SRDS config doesn't work and multiple HCM share the same ``scope_key_builder``.
+- area: logging
+  change: |
+    GRPC_STATUS_NUMBER will not be showed in access_log, for non-grpc request.
```
Comment (yanavlasov) [inline changelogs/current.yaml:83]:
```suggestion
    Do not display GRPC_STATUS_NUMBER for non gRPC requests.
```

**Review by yanavlasov — CHANGES_REQUESTED** [review]
LGTM with spelling suggestion.

**Pull request merged**